docs/Automation.md

```diff
@@ -188,78 +210,41 @@ Sample `C#` code—this is interactive, but yours doesn’t need to be:
-                char[] delimiters = {'='};
-                foreach (string arg in args)
+                string testAppPath = Path.GetFullPath(@"..\myApplication.exe");
+                string outputDir = Path.GetFullPath(@".\TestOutput");
+                Process testProcess = Process.Start(testAppPath);
+                var config = Config.Builder.ForProcessId(testProcess.Id)
+                    .WithOutputFileFormat(OutputFileFormat.A11yTest)
+                    .WithOutputDirectory(outputDir)
+                    .Build();
+
+                var scanner = ScannerFactory.CreateScanner(config);
+
+                try
                 {
-                    string[] pieces = arg.Split(delimiters);
-                    if (pieces.Length == 2)
-                    {
-                        string key = pieces[0].Trim();
-                        string value = pieces[1].Trim();
-                        if (!string.IsNullOrWhiteSpace(key) && !string.IsNullOrWhiteSpace(value))
-                        {
-                            // Special case for SecondaryConfigFile
-                            if (key.Equals("SecondaryConfigFile", StringComparison.OrdinalIgnoreCase))
-                            {
-                                secondaryConfigFile = value;
-                            }
-                            else
-                            {
-                                parameters[key] = value;
-                            }
-                            continue;
-                        }
-                    }
-                    Console.WriteLine("Ignoring malformed input: {0}", arg);
-                };
-                Console.WriteLine(StartCommand.Execute(parameters secondaryConfigFile).ToString());
-                int autoFileId = 0;
-                while (true)
+                    var output = scanner.Scan();
+                    Assert.IsTrue(output.ErrorCount == 0);
+                }
+                catch(AxeWindowsAutomationException e)
                 {
-                    Console.Write("Enter process ID to capture (blank to exit): ");
-                    string input = Console.ReadLine();
-                    if (input == string.Empty)
-                        break;
-                    if (!int.TryParse(input, out int processId))
-                    {
-                        Console.WriteLine("Not a valid int: " + input);
-                        continue;
-                    }
-                    Dictionary<string, string> snapshotParameters = new Dictionary<string, string>
-                    {
-                        { CommandConstStrings.TargetProcessId, input },
-                        { CommandConstStrings.OutputFile, autoFileId++.ToString() },
-                    };
-                    Console.WriteLine(SnapshotCommand.Execute(snapshotParameters).ToString());
+                    var errorMessage = e.InnerException != null ? e.InnerException.Message : e.Message;
```

Comment:
There's at least 1 place (Line 23 of TargetElementLocator.cs) where this might return a less-than-helpful message. Would it make sense to just ToString() the whole Exception? That automatically walks the Exception chain, displaying messages, stack traces, etc.
```suggestion
                    var errorMessage = e.ToString();
```